docs/lib/storage/fragments/flutter/download.md

```diff
@@ -10,3 +10,20 @@ try {
   print(e.toString());
 }
 ```
+
+### Generate a download URL
+
+You can also retrieve a URL for the object in storage:
+
+```dart
+
+try {
+  GetUrlResult result =
+    await Amplify.Storage.getUrl(key: "myKey");
+  let url = result.url; 
```

Comment:
Lets remove this line and instead print `result.url`?

Reply:
ok